lib/rspec/support/reentrant_mutex.rb

```diff
@@ -1,3 +1,5 @@
+require 'fiber'
```

Comment:
I foresee that @JonRowe would say that requiring a dependency from stdlib is something that we try to avoid at all costs.

Reply:
Fair enough.

Reply:
Yes, we don't require things that developers need to require because it causes weird failures for developers. (e.g. RSpec will pass because we require it but it won't work in dev/production).